definitions/infra-awsecsinstance/definition.yml

```diff
@@ -0,0 +1,19 @@
+domain: INFRA
+type: AWSECSCONTAINERINSTANCE
```

Comment:
The folder name needs to be {domain}-{type} so in your case `infra-awsecscontainerinstance` I think

Reply:
Thank you! Missed that part. Pushed fix